utils/gen-unicode-data/Sources/GenScalarProps/Names.swift

```diff
@@ -159,6 +159,12 @@ func emitScalarNames(
     }
   }
   
+  result += """
+  #define NAMES_LAST_SCALAR_OFFSET \(nameBytes.count)
```

Comment:
Normally in C I would define these in terms of the collection:
```
static const size_t _swift_stdlib_names_last_offset = sizeof _swift_stdlib_names / sizeof _swift_stdlib_names[0];
```

Given that they're programmatically generated, I'm ok with doing it they way you have it, so no need to fix this, but I generally prefer this style.